NOTICE

```diff
@@ -1,110 +1,94 @@
-License: BSD-3-clause
+~~~
 
-Files: src/third_party/chromium/src/base/third_party/dmg_fp/*
-Copyright: 1991, 2000, 2001 by Lucent Technologies
-License: dtoac
+Notice Apache HTTP Server (2.4):
 
-Files: src/third_party/chromium/src/base/third_party/dynamic_annotations/*
-Copyright: 2008-2009 Google Inc.
-License: BSD-variant
+Portions of this software were developed at the National Center
+for Supercomputing Applications (NCSA) at the University of
+Illinois at Urbana-Champaign.
 
-Files: src/third_party/chromium/src/base/third_party/icu/*
-       src/third_party/chromium/src/base/i18n/icu_string_conversions.cc
-Copyright: 1995-2009 International Business Machines Corporation and others
-License: ICU
+This software contains code derived from the RSA Data Security
+Inc. MD5 Message-Digest Algorithm, including various
+modifications by Spyglass Inc., Carnegie Mellon University, and
+Bell Communications Research, Inc (Bellcore).
 
-Files: src/third_party/chromium/src/base/third_party/nspr/*
-Copyright: 1998-2000 Netscape Communications Corporation
-           2011 Google Inc.
-License: MPL-1.1 or GPL-2+ or LGPL-2.1+
+~~~
 
-Files: src/third_party/chromium/src/base/third_party/superfasthash/*
-Copyright: 2010 Paul Hsieh
-License: BSD-3-clause
+Notice Apache Portable Runtime:
 
-Files: src/third_party/chromium/src/base/third_party/symbolize/*
-Copyright: 2006 Google Inc.
-License: BSD-3-clause
+Portions of this software were developed at the National Center
+for Supercomputing Applications (NCSA) at the University of
+Illinois at Urbana-Champaign.
 
-Files: src/third_party/chromium/src/base/third_party/valgrind/*
-Copyright: 2000-2010 Julian Seward
-License: libbz2
+This software contains code derived from the RSA Data Security
+Inc. MD5 Message-Digest Algorithm.
 
-Files: src/third_party/chromium/src/googleurl/*
-Copyright: 2007 Google Inc.
-License: BSD-3-clause
+This software contains code derived from UNIX V7, Copyright(C)
+Caldera International Inc.
 
-Files: src/third_party/chromium/src/googleurl/src/url_parse.cc
-Copyright: 1998 Netscape Communications Corporation
-License: MPL-1.1 or GPL-2+ or LGPL-2.1+
+~~~
 
-Files: src/third_party/closure_library/*
-Copyright: 2006-2013 The Closure Library Authors
-License: Apache-2.0
+Notice Apache Serf:
 
-Files: src/third_party/domain_registry_provider/*
-Copyright: 2011 Google Inc.
-License: Apache-2.0
+This product includes software developed by many people, and distributed
+under Contributor License Agreements to The Apache Software Foundation
+(http://www.apache.org/).  See the revision logs for an exact contribution
+history.
 
-Files: src/third_party/domain_registry_provider/src/third_party/effective_tld_names/*
-Copyright: 2007–16 Mozilla Foundation
-License: MPL-2.0
+~~~ 
 
-Files: src/third_party/google-sparsehash/*
-Copyright: 2005 Google Inc.
-License: BSD-3-clause
+Notice utf
 
-Files: src/third_party/libwebp/*
-Copyright: 2010 Google Inc.
-License: BSD-3-clause
+/*
```

Comment:
It's sufficient to keep this part just in the LICENSE file.

Reply:
removed this here.